CoreFoundation/Base.subproj/CFBase.h

```diff
@@ -142,6 +142,18 @@
     #else
         #define CF_EXPORT _CF_EXTERN
     #endif
+#elif TARGET_OS_MAC || TARGET_OS_LINUX || TARGET_OS_BSD || TARGET_OS_ANDROID
+
+#if 0
+#if defined(CoreFoundation_EXPORTS)
```

Comment:
?

Reply:
This is really what the code *should* be, but, for staging purposes, we don't want to hide the symbols in static builds just yet.